app/views/users/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>LIST OF USERS </h1>
\ No newline at end of file
+
+<ul class="users_container">
+  <% @users.each do |user| %>
+    <%= link_to user_path(user.id) do %>
+      <li>
+        <div class="user_img">
+          <img src="<%= user.Photo %>" alt="Photo">
+        </div>
+        <div class="user_details">
+          <h2><%= user.Name %></h2>
+          <p>Number of Posts: <%= user.Posts_counter.nil? ? '0' : user.Posts_counter %></p>
+        </div>
+      </li>
+    <% end %>
+  <% end %>
```

Comment:
- [ ] (Optional) Using partials is a better option as a practice instead of using `@objects.each`. Consider it for future projects